app/assets/stylesheets/views/_homepage.scss

```diff
@@ -159,3 +159,87 @@
   @include govuk-font(36, $weight: bold);
   color: govuk-colour("white");
 }
+
+.homepage-services-and-info {
+  border-bottom: $govuk-focus-width solid govuk-colour("black");
+  padding-bottom: govuk-spacing(7);
+  padding-top: govuk-spacing(6);
+
+  @include govuk-media-query($from: desktop) {
+    padding-bottom: govuk-spacing(6);
+  }
+}
+
+.homepage-services-and-info__list {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```

Comment:
Could we add a comment explaining why we're doing this hard-coded row thing, maybe even a comment like we have in the super nav translation yml in the equivalent in frontend encouraging folks to update the settings here on the rare chance a new topic needs ot be added to the homepage?

Reply:
A note is a good idea. I need to replace this with a mixin for backwards compatibility, so that should give opportunity to document this better.

It's a bit ugly, but it does the job. Unfortunately `repeat()` hasn't got quite the support needed yet.

Reply:
I've updated the `columns()` mixin from the gem in https://github.com/alphagov/frontend/pull/3033/commits/6e86a62ccca9f658b2b7067e7f9006fddbf5c0a1 - in the interests of speed I've added the mixin here for review, but I'll add the updated version to the gem in the future and then use that version here.